packages/web-app-files/src/components/SideBar/FileInfo.vue

```diff
@@ -34,7 +34,6 @@ export default {
   components: {
     PrivateLinkItem
   },
-  inject: ['displayedItem'],
```

Comment:
What exactly is/was the issue? Is the displayedItem not updating properly, or is the usage within this component not reactive?

Reply:
Because `displayedItem` is only a copy of the highlighted file: https://github.com/owncloud/web/blob/c8f8589bf6d66353dd004807872ab3cfb66c7a80/packages/web-app-files/src/components/SideBar/SideBar.vue#L244